From: Antonio Valentino <antonio.valentino@tiscali.it>
Date: Sun, 17 Nov 2019 10:15:09 +0000
Subject: Clean fixtures

Forwarded: https://github.com/zarr-developers/numcodecs/pull/249
---
 numcodecs/tests/common.py | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/numcodecs/tests/common.py b/numcodecs/tests/common.py
index bb7c478..a25362c 100644
--- a/numcodecs/tests/common.py
+++ b/numcodecs/tests/common.py
@@ -311,6 +311,10 @@ def check_backwards_compatibility(codec_id, arrays, codecs, precision=None, pref
                     assert_array_equal(arr, dec_arr)
                     assert arr_bytes == ensure_bytes(dec)
 
+    if os.path.exists(fixture_dir):
+        import shutil
+        shutil.rmtree(fixture_dir)
+
 
 def check_err_decode_object_buffer(compressor):
     # cannot decode directly into object array, leads to segfaults
