From: Antonio Valentino <antonio.valentino@tiscali.it>
Date: Sat, 21 Nov 2020 19:20:43 +0000
Subject: Cpuinfo

Forwarded: https://github.com/zarr-developers/numcodecs/pull/258
---
 setup.py | 13 +++++++++----
 1 file changed, 9 insertions(+), 4 deletions(-)

diff --git a/setup.py b/setup.py
index 46a0aa0..d25f58c 100644
--- a/setup.py
+++ b/setup.py
@@ -2,14 +2,19 @@ import os
 import sys
 from glob import glob
 
-import cpuinfo
 from Cython.Distutils.build_ext import new_build_ext as build_ext
 from setuptools import Extension, setup
 from setuptools.errors import CCompilerError, ExecError, PlatformError
 
-# determine CPU support for SSE2 and AVX2
-cpu_info = cpuinfo.get_cpu_info()
-flags = cpu_info.get('flags', [])
+try:
+    # determine CPU support for SSE2 and AVX2
+    import cpuinfo
+    cpu_info = cpuinfo.get_cpu_info()
+    flags = cpu_info.get('flags', [])
+except Exception:
+    # cpuinfo not available or does not support the architecture/os
+    flags = []
+
 have_sse2 = 'sse2' in flags
 have_avx2 = 'avx2' in flags
 disable_sse2 = 'DISABLE_NUMCODECS_SSE2' in os.environ
