From: Antonio Valentino <antonio.valentino@tiscali.it>
Date: Sat, 21 Nov 2020 18:07:18 +0000
Subject: Skip broken tests on BE architectures

Forwarded: https://github.com/zarr-developers/numcodecs/issues/256
---
 numcodecs/tests/test_categorize.py | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/numcodecs/tests/test_categorize.py b/numcodecs/tests/test_categorize.py
index 0ec115b..8fb51ee 100644
--- a/numcodecs/tests/test_categorize.py
+++ b/numcodecs/tests/test_categorize.py
@@ -73,6 +73,8 @@ def test_repr():
     assert expect == actual
 
 
+import sys
+@pytest.mark.skipif(sys.byteorder == 'big', reason='broken test no BE architectures')
 def test_backwards_compatibility():
     codec = Categorize(labels=labels, dtype='<U4', astype='u1')
     check_backwards_compatibility(Categorize.codec_id, arrays, [codec], prefix='U')
